spyder/app/utils.py

```diff
@@ -311,6 +311,7 @@ def create_window(WindowClass, app, splash, options, args):
         QCoreApplication.setAttribute(Qt.AA_DontShowIconsInMenus, True)
 
     # Open external files with our Mac app
+    # ??? Do we need this?
     if running_in_mac_app():
```

Comment:
If the Spyder app is still registering with Mac to open files in `info.plit`, I think this needs to stay.

Reply:
I'm not doing anything with that right now. This was a note to revisit later.

Reply:
I understand that, but since you asked if it needs to stay, I'm just answering that I think it does.

Reply:
I agree, I think this is needed if you want to open a file on mac by double clicking it or drag and drop on the spyder icon